pychromecast/discovery.py

```diff
@@ -128,3 +137,19 @@ def callback(name):
     finally:
         if browser is not False:
             stop_discovery(browser)
+
+
+def get_host_from_service(service, zconf):
+    """ Resolve host and port from service. """
+    host_info = (None, None)
+    try:
+        service_info = zconf.get_service_info('_googlecast._tcp.local.',
+                                              service)
+        if service_info and service_info.server and service_info.port:
+            _LOGGER.debug(
+                "get_host_from_service resolved service %s to service_info %s",
+                service, service_info)
+            host_info = (service_info.server.lower(), service_info.port)
```

Comment:
Any reason we wouldn't just return `service_info`  ?

Reply:
And then just return either `None` or raise when fetching fails.

Reply:
Fixed.